docs/docker-images/nodejs.md

```diff
@@ -0,0 +1,27 @@
+# Node.js
+
+The [Lagoon `node` Docker image](https://github.com/amazeeio/lagoon/blob/main/images/node/Dockerfile). Based on [the official PHP Alpine images](https://hub.docker.com/_/php/).
+
+## Supported Versions
+
+* 10 - End of Life as of 2021-04-30
+* 12
+* 14
+
+{% hint style="info" %}
+We stop updating end of life Node.js images usually with the lagoon release after the officialy communicated End of life date https://nodejs.org/en/about/releases/
+{% endhint %}
+
+## Lagoon adaptions
+We ship 2 versions of Node.js images the normal node:version image and the node:version-builder
+
+The builder variant of those images comes with additional tooling that is needed when you run node.js builds for a full list check out the [Dockerfile](https://github.com/amazeeio/lagoon/blob/main/images/node/builder/Dockerfile)
```

Comment:
```suggestion
The builder variant of those images comes with additional tooling that is needed when you run node.js builds for a full list check out the [Dockerfile](https://github.com/uselagoon/lagoon-images/blob/main/images/node/builder/Dockerfile)
```

We probably need to update all those Dockerfile links :-(

Reply:
oh yes - i'd basically queue that in as soon as we got the images over there to have just a commit-set that fixes up those links